CMakeLists.txt

```diff
@@ -585,7 +579,7 @@ endif()
 set(WAYLAND_FEATURE_PURPOSE "Wayland")
 set(WAYLAND_FEATURE_DESCRIPTION "Wayland client")
 
-set(ZLIB_FEATURE_TYPE "REQUIRED")
+set(ZLIB_FEATURE_TYPE "RECOMMENDED")
```

Comment:
better use `find_library` if required and remove the `find_feature` (only used for emulation anyway)